app/build.gradle.kts

```diff
@@ -86,9 +83,11 @@ dependencies {
 
     coreLibraryDesugaring(libs.android.desugaring)
 
-    // Compose Destinations
-    implementation(libs.compose.destinations)
-    ksp(libs.compose.destinations.ksp)
+    // Safer Navigation Compose
+    implementation(libs.safer.navigation.compose.animation)
```

Comment:
libs.compose.saferNavigation might read nicer

Reply:
i'm just following the current convention

Reply:
We don't have a fixed convention for this

Reply:
We do actually, the strategy I use is to only create an accessor if the one before it is reused. So if we don't have a `libs.safer.*` then use camelCase.